### PR TITLE
Update 5.7.0

### PR DESCRIPTION
**Pull request by Jrice1317**
### Changes made
- Updated version and sha256
- Updated requirements
- Added description

1. Jira Ticket: [PKG-157](https://anaconda.atlassian.net/browse/PKG-157)
2. Upstream: https://github.com/jupyter/nbformat/tree/5.7.0
3. Change log: https://nbformat.readthedocs.io/en/latest/changelog.html

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,11 +20,12 @@ build:
...
   host:
     - python
-    - flit-core >=3.2,<4
     - pip
-    - wheel
+    - hatchling
+    - hatch-nodejs-version
   run:
     - python
+    - importlib_metadata >=3.6
```
Comment (cbouss) [inline recipe/meta.yaml:28]: You could set this instead: `- importlib_metadata >=3.6. # [py<38]` as this only applied to python lower than 3.8.
https://github.com/jupyter/nbformat/blob/5.7.0/pyproject.toml#L29

**Review by cbouss — COMMENTED** [review]
Good to merge after the selector is added to importlib_metadata

File: recipe/meta.yaml (hunk 1 above)
Comment (Jrice1317) [reply recipe/meta.yaml:28]: Okay, thanks!

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Review by cbouss — APPROVED** [review]

**Pull request merged**